python/pyspark/sql/functions/builtin.py

```diff
@@ -3685,7 +3685,7 @@ def regr_avgx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           2.75|  2.75|
     +---------------+------+
 
-    Example 2: All paris's x values are null
+    Example 2: All pairs's x values are null
```

Comment:
let's be consistent:
```suggestion
    Example 2: All pairs' x values are null
```